### PR TITLE
Update the "Managing Users" space docs

### PR DESCRIPTION
**Pull request by rememberlenny**
Adding documentation for the space user management.

### CODE REVIEW

**Comment by brittag** [comment]
Would you be up for splitting out the content change and the CSS/Javascript change into two PRs? It sounds like they could be reviewed independently of each other. :)

**Comment by mogul** [comment]
The markup would be a little more semantic if the shortcode was `{{% alternatives %}}` instead of `{{% tabs %}}`... We might choose to render it differently for mobile, print, etc. where we don't actually want to render with tabs.

**Comment by rememberlenny** [comment]
@brittag This PR has been updated to reflect a _content only_ change to the documentation.

Updated PR with the new shortcode name is here: https://github.com/18F/cg-site/pull/1020 @mogul

File: content/docs/apps/managing-teammates.md [hunk 1]
```diff
@@ -23,18 +23,30 @@ Org Managers can assign roles using the [dashboard](https://dashboard.fr.cloud.g
...
-To give a teammate access to a space, go to your org page, click a space name, and scroll down to "Space users". You can add a person, and then you can give them roles with the checkboxes.
+### Space users
+
+If you're an Space Manager, go to your [dashboard](https://dashboard.fr.cloud.gov/) and click on the name of your org, then click on the name of your space.
+
+![Space roles](/img/space-user-role-management.png)
+
+On the space page, scroll down to "Space users". This lists the space users and their space roles. Change the roles by clicking the checkboxes beside their names. (If you're the only person in the space, you'll just see "You are the only user in this space".)
+
+To give a organization user access to a space, go to your space page, click a space name, and scroll down to "Space users". You can add a person, and then you can give them roles with the checkboxes.
```
Comment (jcscottiii) [inline content/docs/apps/managing-teammates.md]: `To give a organization user` -> `To give an organization user` or `To give an existing organization user`

Comment (rememberlenny) [reply content/docs/apps/managing-teammates.md]: 👍

**Review by gboone — APPROVED** [review]

File: content/docs/apps/managing-teammates.md [hunk 2]
```diff
@@ -23,18 +23,30 @@ Org Managers can assign roles using the [dashboard](https://dashboard.fr.cloud.g
...
 If you're an Org Manager, go to your [dashboard](https://dashboard.fr.cloud.gov/) and click on the name of your org.
 
 ![User roles](/img/user-role-management.png)
 
 On the org page, scroll down to "Organization users". This lists the org users and their org roles. Change the roles by clicking the checkboxes beside their names. (If you're the only person in the org, you'll just see "You are the only user in this organization".)
 
-To give a teammate access to a space, go to your org page, click a space name, and scroll down to "Space users". You can add a person, and then you can give them roles with the checkboxes.
+### Space users
+
+If you're an Space Manager, go to your [dashboard](https://dashboard.fr.cloud.gov/) and click on the name of your org, then click on the name of your space.
```
Comment (brittag) [inline content/docs/apps/managing-teammates.md:36]: Typo here - "an Space Manager" instead of "a Space Manager"

**Pull request merged**